pkg/oam/util/helper_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"github.com/crossplane/oam-controllers/pkg/oam/util"
 )
 
```

Comment:
How can I run `helper_test.go`?  Please help advise.
`go test help_test.go` won't work with warning `ok  	command-line-arguments	0.702s [no tests to run]` and nor cloud Goland run it.

Reply:
ginkgo -v .